app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -72,6 +74,7 @@ class StateFragmentPresenter @Inject constructor(
   private lateinit var currentStateName: String
   private lateinit var binding: StateFragmentBinding
   private lateinit var recyclerViewAdapter: RecyclerView.Adapter<*>
+  private val splitScreenManager: SplitScreenManager = SplitScreenManager(activity)
```

Comment:
You can inject this class directly once the class is converted to dagger class.

Reply:
Done